app/models/post/status.rb

```diff
@@ -1,36 +1,16 @@
 module Post::Status
   extend ActiveSupport::Concern
 
-  ACTIVE = 0
-  COMPLETE = 1
-  HIATUS = 2
-  ABANDONED = 3
-
   included do
-    def completed?
-      status == COMPLETE
-    end
+    enum status: {
+      active: 0,
+      complete: 1,
+      hiatus: 2,
+      abandoned: 3,
+    }
 
     def on_hiatus?
-      marked_hiatus? || (active? && tagged_at < 1.month.ago)
-    end
-
-    def marked_hiatus?
-      status == HIATUS
-    end
-
-    def active?
-      status == ACTIVE
-    end
-
-    def abandoned?
-      status == ABANDONED
+      hiatus? || (active? && tagged_at < 1.month.ago)
     end
   end
-
-  def self.get_status(param)
```

Comment:
We didn't seem to be using this anywhere? Did I miss it?

Reply:
I think when status is set?